### DISCUSSION TITLE
Placing operators at the beginning of lines (operator-linebreak, same issue from `prettier`)

### DISCUSSION THREAD

**Discussion by seahindeniz** [biomejs/biome#1693] [Ideas]
Hey Biome folks,

~~Just stumbled upon an issue in Prettier that's been hanging around for a while (like 5-6 years).
Here's the deal: [Issue #3806](https://github.com/prettier/prettier/issues/3806).~~
Prettier just merged the related PR https://github.com/prettier/prettier/pull/7111

Also, ESLint has this for a long time https://eslint.org/docs/latest/rules/operator-linebreak

Seems like something you might find interesting. Any chance you could take a peek and see if there's something we can do about it with Biome?

Thanks a bunch!

**Comment by dmythro** [comment]
Oh, yes! I'm wondering why it is not a feature yet! So. many people are annoyed by not having this in Prettier, and I am amongst them :) 
Please?

**Comment by artrz** [comment]
I second this. Having operators at the beginning of lines makes code easier to read and understand, reducing the cognitive overload.

**Comment by NexZhu** [comment]
+1 need this!
The ESLint rule has moved to:
https://eslint.style/rules/default/operator-linebreak

**Comment by tr-serhii** [comment]
+1

**Comment by Psarsy** [comment]
+1000

**Comment by seahindeniz** [comment]
@arendjr sorry for tagging but just to heads up, Prettier just merged the [related PR](https://github.com/prettier/prettier/pull/7111) and it's planned for v4 https://github.com/prettier/prettier/milestone/42?closed=1

May I ask, is the same rule going to be planned for Biome?

**Reply by arendjr** [reply]
We do strive for Prettier compatibility, so I think we’ll eventually implement this too. No ETA though, so any help is appreciated!

**Comment by webda2l** [comment]
:+1:  An equivalent of prettier 3.5 `experimentalOperatorPosition` (https://prettier.io/blog/2025/02/09/3.5.0#javascript) for follow-up.

**Comment by ematipico** [comment]
This has been implemented https://github.com/biomejs/biome/pull/6159 it will be available in the next minor
